packages/toolpad-app/src/config.ts

```diff
@@ -65,11 +63,11 @@ function getBrowsersideRuntimeConfig(): RuntimeConfig {
   return maybeRuntimeConfig;
 }
 
-export const runtimeConfig: RuntimeConfig =
+const runtimeConfig: RuntimeConfig =
   typeof window === 'undefined'
     ? {
         // Define runtime config here
-        foo: process.env.TOOLPAD_FOO,
+        integrationTest: !!process.env.TOOLPAD_INTEGRATION_TEST,
```

Comment:
was something like this what i was missing for CI to work?

Reply:
We were using built-time configuration for this, and we build the docker image for production, so it was set to false